files/image_config/monit/memory_checker

```diff
@@ -140,10 +140,10 @@ def get_running_container_names():
         running_container_list = docker_client.containers.list(filters={"status": "running"})
         running_container_names = [ container.name for container in running_container_list ]
     except (docker.errors.APIError, docker.errors.DockerException) as err:
-        syslog.syslog(syslog.LOG_ERR,
```

Comment:
>LOG_ERR [](http://example.com/codeflow?start=29&length=7)

Is it possible to check first docker service is running, if Failed to retrieve the running container list, treat it still as ERR?

Reply:
That is the idea of `get_running_container_names()`, if the `docker service is not running it just gives us a warning message to the syslog and returns the empty list of docker containers.

Reply:
To clarify:
1. check docker service is running, if not running, no ERR, no WARNING. Low frequent INFO may be good.
2. if docker service is running, retrieve the running container list. If failed to retrieve, treat it still as ERR.

Reply:
We already have the check if the `docker service is running, but in some cases (which I described in the PR description) is not enough.
This check is right before the `get_running_container_names()` function call.

Reply:
Thanks for the explanation! Let me update my proposal:
1. when you catch an exception, if you are sure docker service is not running, log INFO in low frequency
2. if not a docker service issue, something really bad happens, let's still use ERR.